b2b-sample/libs/business-admin-app/data-access/data-access-common-models-util/src/lib/identityProvider/identityProviderUtils.ts

```diff
@@ -17,10 +17,30 @@
  */
 
 import { getManagementAPIServerBaseUrl, getOrgUrl } from "@b2bsample/shared/util/util-application-config-util";
-import { ENTERPRISE_ID, GOOGLE_ID } from "@b2bsample/shared/util/util-common";
+import { EMPTY_STRING, ENTERPRISE_ID, GOOGLE_ID } from "@b2bsample/shared/util/util-common";
 import IdentityProviderDiscoveryUrl from "./identityProviderDiscoveryUrl";
 import IdentityProviderTemplateModel from "./identityProviderTemplateModel";
-import config from "../../../../../../../config.json";
+import enterpriseImage from "../../../../../ui/ui-assets/src/lib/images/enterprise.svg";
+import googleImage from "../../../../../ui/ui-assets/src/lib/images/google.svg";
+
+/**
+ * @param templateId - template id of the identity provider
+
+ * @returns - local image for the relevant identity provider
+ */
+export function getImageForTheIdentityProvider(templateId: string): string {
+    if (GOOGLE_ID === templateId) {
+
+        return googleImage;
+    }
+    if (ENTERPRISE_ID === templateId) {
+
+        return enterpriseImage;
+
```

Comment:
Let's remove the new line after the return statement to make it consistent with other places.

Reply:
Fixed with https://github.com/wso2/samples-is/pull/416/commits/e238027e3ab758576ae189fd6ec3d1496abb6315